controllers/reviews.js

```diff
@@ -0,0 +1,9 @@
+const { selectAllReviews } = require("../models/reviews");
+
+exports.getAllReviews = (req, res, next) => {
+  selectAllReviews()
+    .then((reviews) => {
+      res.status(200).send(reviews);
```

Comment:
we always want to be sending our response back as an object with a descriptive key in our controller i.e. .send({reviews: response}) - your tests may need a slight refactor to compensate

Reply:
ok I will change this